.golangci.yml

```diff
@@ -212,6 +212,9 @@ linters:
     # We allow cuddling assignment following conditions because there are valid 
     # logical groupings for this use-case (e.g. when evaluating config values).
     - wsl
+    # New line required before return would require a large fraction of the
+    # code base to need updating, it's not worth the perceived benefit.
+    - nlreturn
```

Comment:
Let's cut this separately, please

Reply:
Sure thing.

Reply:
Opened https://github.com/m3db/m3/pull/2865.

Reply:
Stamped, thanks!